internal/cli/deployment_list.go

```diff
@@ -10,19 +10,15 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/golang/protobuf/ptypes"
-	"github.com/posener/complete"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
-	"google.golang.org/protobuf/proto"
-	"google.golang.org/protobuf/types/known/anypb"
-
-	sdk "github.com/hashicorp/waypoint-plugin-sdk/proto/gen"
 	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	clientpkg "github.com/hashicorp/waypoint/internal/client"
 	"github.com/hashicorp/waypoint/internal/clierrors"
 	"github.com/hashicorp/waypoint/internal/pkg/flag"
 	pb "github.com/hashicorp/waypoint/internal/server/gen"
 	serversort "github.com/hashicorp/waypoint/internal/server/sort"
+	"github.com/posener/complete"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
These should move back up to line 13-15

Reply:
Yep, more background for @izaaklauer if you aren't aware of our style, we do:

```go
import(
  <stdlib stuff>

  <3rd party stuff>

  <internal stuff>
)
```

Reply:
Ah ok. My IDE has a mind of it's own when it comes to import ordering - i'll wrangle it into matching our convention.